packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -778,6 +778,12 @@ const conf = (module.exports = convict({
     env: 'SOURCE_MAP_TYPE',
     format: String,
   },
+  webpackModeOverride: {
+    default: undefined,
+    doc: 'Tells webpack how to optimize build. See https://webpack.js.org/configuration/mode/',
+    env: 'WEBPACK_MODE_OVERRIDE',
+    format: String,
```

Comment:
Q: Should this be String, or just explicitly list valid values `['production', 'development', 'none']`?